src/__tests__/reducers/reducers.test.js

```diff
@@ -0,0 +1,246 @@
+import {calculate} from '../../reducers/calculator';
+import calculator from '../../reducers/calculator';
+import * as operators from "../../utils/actionTypes";
+import {onClearClick,onNumClick,onEqualClick,onOpeClick} from '../../actions/index';
+ 
+describe('Calculate function',()=>{
```

Comment:
indention is wrong here.
everything after describe should be indented.
you could also try auto format of VSCode - https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode

Reply:
I will fix that, I will use this option that you show me.